src/providers/FileSystemProvider/FileSystemProvider.ts

```diff
@@ -864,11 +859,6 @@ export class FileSystemProvider implements vscode.FileSystemProvider {
   // Fetch entry (a file or directory) from cache, else from server
   private async _lookup(uri: vscode.Uri, fillInPath?: boolean): Promise<Entry> {
     const api = new AtelierAPI(uri);
-    if (uri.path === "/") {
```

Comment:
What's being removed here seems to relate to my fix for #695 from 4 years ago. IIRC that issue could arise if I create XYZ.code-workspace with a folder whose isfs uri references a server definition name that's in my settings.json, then I send you the workspace file to use but you don't have the server definition set up. Rather than failing silently this code raises a message.

Do the benefits of removing it in this PR outweigh those of retaining it?

Reply:
Yes, removing this line did allow that bug to reappear. I've found another way to fix it, see my latest commit.